### ISSUE TITLE
Remove the «Create a groupchat» item until groups are working

### ISSUE THREAD

**Issue by SkyzohKey** [Antox/Antox#435]
Leaving it as it may lead users to deception. Removing the options until groupchats are not implemented seems to be the best option. Else the user might think the app is incomplete (ok it is but then...)

**Comment by wiiaam** [comment]
Yeah could do, since tox4j probably wont have groupchats for a wee while
